transformer_engine/common/__init__.py

```diff
@@ -26,8 +26,14 @@ def is_package_installed(package):
     )
 
 
-def get_te_path():
+def get_te_path() -> Path:
     """Find Transformer Engine install path using pip"""
+
+    # If TE is installed in the main python package directory, prioritize it
+    # for cases where user is running from a TE directory where an editable
+    # build is not installed.
+    if (Path(sysconfig.get_path("purelib")) / "transformer_engine").exists():
+        return Path(sysconfig.get_path("purelib"))
```

Comment:
This is helpful for developers making pure Python changes, but it's inconvenient for my C++ workflow since you can't just change the local `.so`s. I think there are ways we could make this nicer (check install dir by default, add envvar to allow manual overload), but it may be better for a separate PR.
```suggestion
```

Reply:
Could you describe the workflow you refer to? If you have a local/editable build, then this change should be a noop as the TE directory would be absent from site-packages.

Reply:
https://xkcd.com/1172/